scripts/ci/run-jenkins.sh

```diff
@@ -17,15 +17,32 @@ if [[ ${CI_TAGS} == *'pull-request'* ]]; then
 	# Skip lanes which are not affected by the PR
 	wget -O pr-contents.diff "${ghprbPullLink}.diff"
 	grep '^diff' pr-contents.diff > pr-files.txt
-    echo "Files affected by the PR:"
+	echo "Files affected by the PR:"
 	cat pr-files.txt
 
 	# FIXME: Add more
+	skip=false
+	skip_step=""
 	if ! grep -q -v a/netcore pr-files.txt; then
-		echo "NetCore only PR, skipping."
-		${TESTCMD} --label="Skipped on NETCORE." --timeout=60m --fatal sh -c 'exit 0'
-        if [[ $CI_TAGS == *'apidiff'* ]]; then report_github_status "success" "API Diff" "Skipped." || true; fi
-        if [[ $CI_TAGS == *'csprojdiff'* ]]; then report_github_status "success" "Project Files Diff" "Skipped." || true; fi
+		skip_step="NETCORE"
+		skip=true
+	fi
+	if ! grep -q -v mini-ppc pr-files.txt; then
```

Comment:
shouldn't it be `a/mono/mini/mini-ppc` ?

Reply:
It could be mini-ppc.h as well.

Reply:
Fixed.